paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/lpms/CashAppLpmTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.paymentelement.confirmation.lpms
+
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+internal class CashAppLpmTest(
```

Comment:
I'm surprised this test passes. Since we're expecting a completed event. But we don't actually complete this payment method, right?

Reply:
Neat! This is a good idea, and probably useful too!